script/bootstrap

```diff
@@ -67,7 +67,17 @@ install_packages_rpm()
 
 install_packages_brew()
 {
-    echo 'macOS not supported currently' && false
+    # add autotools
+    brew install automake libtool
+
+    # add ARM toolchain
+    brew tap PX4/homebrew-px4
+    brew install gcc-arm-none-eabi
```

Comment:
I believe this install [v5.4](https://github.com/PX4/homebrew-px4/blob/master/gcc-arm-none-eabi.rb#L7) but that may change overtime. Should this install a specific version?

Reply:
We intentionally do not prescribe a particular release.  On [Travis](https://travis-ci.org/openthread/openthread), we test against the 4, 5, and 6 series.

Reply:
Fair, only thing is it may result in more issues :)